plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryUtil.java

```diff
@@ -82,4 +83,9 @@ public static String quote(String name)
     {
         return QUOTE + name.replace(QUOTE, ESCAPED_QUOTE) + QUOTE;
     }
+
+    public static String quoted(RemoteTableName table)
```

Comment:
nit: `quote` above vs `quoted` here looks kinda unnatural. I would keep name the same.